sycl/doc/FAQ.md

```diff
@@ -119,9 +119,9 @@ specification.
 
 ## Device specific questions and issues
 
-### Q: What devices are supported by Intel SYCL compiler?
-**A:** By design, SYCL is closely connected to OpenCL, which is used to offload
-code to specialized devices. Intel SYCL compiler currently makes use of SPIR-V,
+### Q: What devices are supported by DPC++ compiler?
+**A:** By design, DPC++ and SYCL are closely connected to OpenCL, which is used
+to offload code to specialized devices. DPC++ compiler currently makes use of SPIR-V,
```

Comment:
Please, keep 80-character limit.
A few place in this document.

Reply:
done, but kept headers and dumps untouched.